src/box/memtx_tx.h

```diff
@@ -539,6 +539,59 @@ memtx_tx_snapshot_clarify(struct memtx_tx_snapshot_cleaner *cleaner,
 	return memtx_tx_snapshot_clarify_slow(cleaner, tuple);
 }
 
+/** Helper of memtx_tx_snapshot_invisible_count_matching. */
+size_t
+memtx_tx_snapshot_invisible_count_matching_until_slow(
```

Comment:
It's a pity that we add dead code. Can we cover it with tests in CE? If not, can we move it to EE? Maybe just put this code under `ifdef ENABLE_READ_VIEW`?

Reply:
Wrapped declarations in the header with the `ENABLE_READ_VIEW` guard.